packages/server/src/api/routes/tests/permissions.spec.ts

```diff
@@ -125,6 +125,12 @@ describe("/permission", () => {
     })
 
     it("should be able to access the view data when the table is set to public and with no view permissions overrides", async () => {
+      // Make view inherit table permissions
+      await config.api.permission.revoke({
+        roleId: STD_ROLE_ID,
+        resourceId: view.id,
+        level: PermissionLevel.READ,
+      })
```

Comment:
I'm a teensy bit confused, let me see if I understand.

Before this PR, views inherit the permissions set on their underlying table. I assume there exists code that does those checks somewhere outside of this PR.

In this PR, on view creation, you are setting permissions on the view explicitly, based on permissions on the table.

What is confusing me is why you need to do these revokes in the tests. I would have thought the permission on the views is identical to how it would have been before your changes? 🤔

Reply:
You're correct. These additional revokes are just needed because the old behaviour was to inherit access, while now it defaults to having explicit permissions. These tests were deliberately testing inheritance, so I need to revoke the explicit permissions in order to continue to test inheritance.